air-quality-backend/scripts/run_in_situ_etl.py

```diff
@@ -15,10 +16,16 @@ def main():
     load_dotenv()
 
     cities = get_locations_by_type(AirQualityLocationType.CITY)
+
+    open_aq_cities = (
+        os.environ["OPEN_AQ_CITIES"] if "OPEN_AQ_CITIES" in os.environ else None
+    )
+    if open_aq_cities is not None:
+        cities = [city for city in cities if city["name"] in open_aq_cities.split(",")]
```

Comment:
Not fundamentally opposed to this, but wouldn't it be better just to create the entries you want inside the collection itself before running the tests? (I assume there will be tests doing this kind of DB setup in places anyway)

Can always run the add cities script again afterwards

Reply:
I did consider that but wasn't sure if rerunning the create cities script from a tests would be a good thing to do, they feel more constant to me. 
I'll take another look during my next PR though (I need to add more tests which might affect how much I need to change the cities DB)